packages/main/src/plugin/message-box.ts

```diff
@@ -15,11 +15,27 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  ***********************************************************************/
+import type { IconDefinition } from '/@api/icon-info.js';
+
 import type { ApiSenderType } from './api.js';
 import { Deferred } from './util/deferred.js';
 
 type DialogType = 'none' | 'info' | 'error' | 'question' | 'warning';
 
+export type RemindOption = 'Remind me tomorrow' | 'Remind me in 2 days' | `Don't show again`;
```

Comment:
I don't see this used, and IMHO shouldn't be part of the message box API - we may want some consistency between remind options in experimental features, but (e.g.) updates could have different times.

(if it did stay, 'Don't show again' just needs regular quotes like the other two)